test/test_tagging.py

```diff
@@ -0,0 +1,109 @@
+import json
+from unittest.mock import (
+    PropertyMock,
+    patch,
+)
+
+from azul.deployment import (
+    populate_tags,
+)
+from azul.logging import (
+    configure_test_logging,
+)
+from azul_test_case import (
+    AzulUnitTestCase,
+)
+
+
+def setupModule():
+    configure_test_logging()
+
+
+class TestAppLogging(AzulUnitTestCase):
+
+    @patch('subprocess.run', new_callable=PropertyMock)
+    def test(self, terraform_mock):
+        # azul.deployment.populate_tags must be tested here because Terraform
+        # is not installed on Travis for unit tests and as such requires mock
```

Comment:
I appreciate this comment, but I think it's assuming some context that isn't obvious. "must be tested **here**" as opposed to where? Having said that, I don't think that's a question worth answering. It's sufficient to say "We mock Terraform since it's not available on Travis".

I am concerned about the mocked response becoming inaccurate if the Terraform response changes. I think it would be helpful to explain here why that's unlikely to be a problem (because this only depends on the provider, not any of the resources, etc).